app/models/parser/instagram_item.rb

```diff
@@ -3,21 +3,22 @@ class InstagramItem < Base
     include ProviderInstagram
 
     INSTAGRAM_ITEM_URL = /^https?:\/\/(www\.)?instagram\.com\/(p|tv|reel)\/([^\/]+)/
+    INSTAGRAM_HOME_URL = /^https?:\/\/(www\.)?instagram\.com\/?$/
     
     class << self
       def type
         'instagram_item'.freeze
       end
 
       def patterns
-        [INSTAGRAM_ITEM_URL]
+        [INSTAGRAM_ITEM_URL, INSTAGRAM_HOME_URL]
       end
     end
     
     private
 
     # Main function for class
-    def parse_data_for_parser(doc, _original_url, _jsonld_array)
+    def parse_data_for_parser(doc, original_url, _jsonld_array)      
```

Comment:
@vasconsaurus , please delete this trialing space before merging :smile:

Reply:
done! 😁